src/WmContentManager.php

```diff
@@ -239,15 +240,10 @@ public function getEntityTeaser(EntityInterface $entity)
     {
         // Allow overrides through event dispatching.
         $event = new WmContentEntityLabelEvent($entity);
-    
-        if (!$return = $entity->label()) {
```

Comment:
Eck returns the title or if there isn't a title field, it'll return `""`

ContentEntityBase does the same except it'll return `null` by default.

Perhaps it's indeed best to default to

```php
$return = $entity->label() ?: "ID: $entity->id()";
```

or something similar